cibuildwheel/__main__.py

```diff
@@ -29,6 +29,17 @@
     resources_dir,
 )
 
+MANYLINUX_ARCHS = (
```

Comment:
I expect order doesn't matter and this could be a set, but I left it as is, so the ordering would match in the dict it creates.